src/page-multi-facility/FacilityPage.tsx

```diff
@@ -27,6 +28,12 @@ const FacilityPage: React.FC = () => {
           localeDataSource={localeDataSource}
         >
           <FacilityPageDiv>
+            {scenario.data && (
+              <ReadOnlyScenarioBanner
+                scenario={scenario.data}
+                dispatchScenarioUpdate={dispatchScenarioUpdate}
+              />
```

Comment:
These properties were available here so I passed them into this new component, but I could have just as easily used `useScenario` within the component itself.  It's still not entirely clear to me when to favor passing properties into a component vs. accessing them via a more global context.

Reply:
I also don't have a great and handy heuristic, but I think this feels like an appropriate use of props. I _think_ in general we want to favor property injection versus the use of hooks for elements that are static, like this banner.